main.go

```diff
@@ -1,18 +1,33 @@
 package main
 
 import (
+	"flag"
 	"log"
 	"net/http"
 	"os"
 
 	"github.com/backpulse/core/database"
 	"github.com/backpulse/core/routes"
 	"github.com/backpulse/core/utils"
+	"github.com/joho/godotenv"
 	"github.com/rs/cors"
 )
 
+var (
+	envFile string
+)
+
+func init() {
+	// env file and will load them into ENV for this process.
+	// will not overload value that defined in ENV had present.
+	flag.StringVar(&envFile, "env", ".env", "env config file")
+}
+
 func main() {
+	flag.Parse()
+
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
+	godotenv.Load(envFile)
```

Comment:
godotenv.Load(...) load config from .env file but will not overload environment value that had defined in system environment